scripts/generate-files.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
```

Comment:
Since this needs to be rewritten for every language, lets use Go here :)

Reply:
It will work for any language and since js and java already using node.js images that would be no brainer. 
Do not like us to maintain 3 different implementations. Only template will be different rest stays the same. 

Golang counterpart will be longer and require golang env that will slow down builds :)

Reply:
Thanks.